crates/holochain/src/conductor/space.rs

```diff
@@ -652,7 +652,10 @@ impl Spaces {
                 Some(t) => t,
                 // If the workflow has not been spawned yet we can't handle incoming messages.
                 // Note this is not an error because only a validation receipt is proof of a publish.
-                None => return Ok(()),
+                None => {
+                    tracing::warn!("No sys validation trigger yet for space: {}", dna_hash);
```

Comment:
It would be nice to know if this is happening, especially in tests. It's only a startup condition but those are most likely to be a problem in tests really.